src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md

```diff
@@ -0,0 +1,36 @@
+
+All four agencies purchased payment acceptance devices manufactured by SC Soft, one of the hardware vendors awarded an MSA by the California Department of General Services (DGS).
+
+For transit processor services—the fare calculation software that allows for pay-as-you-go “fare capping,” enabling transit customers to pay as they go up to “capped” amounts equivalent to the cost of a daily, weekly, or monthly pass—the transit agencies contracted with Littlepay. Two of the four agencies have already enabled fare capping, with HTA providing riders with daily, weekly, and monthly capping and MTA activating daily capping on its routes.
+
+And for payment processing—the secure back-end service that transmits fares from riders’ tapped bank cards and smart devices to transit providers’ bank accounts—the agencies contracted with Elavon, a company with which the State of California has an MSA for Electronic Payment Acceptance Services (EPAY) for California-based government agencies.
+In addition to enabling riders to tap to pay on fixed-route buses, the agencies are the first in the state to roll out open-loop payments on their dial-a-ride services. Through a six-month demonstration project with Felix Payment Systems, the agencies introduced open-loop digital payments on their dial-a-ride services by allowing their riders to tap their contactless bank cards or mobile wallets on NFC-enabled tablets.
+
+## About Cal-ITP
+The California Integrated Travel Project (Cal-ITP) was established by the California State Transportation Agency (CalSTA) and California Department of Transportation (Caltrans) to both improve and encourage the use of multimodal travel throughout California—by enabling contactless open-loop payments, standardizing information for easy multimodal trip planning, and automating customer discounts ([Cal-ITP Benefits](https://www.calitp.org/assets/Cal-ITP.Benefits.PressRelease.220921.pdf?utm_source=Press+Release&utm_medium=Website&utm_id=FNG)). Over the past two years, Cal-ITP successfully led contactless open-loop contactless payment implementations in California on Monterey-Salinas and Santa Barbara buses; Sacramento light rail; intercity passenger rail between Sacramento and the San Francisco Bay Area; on-demand transit vans; and LAX’s FlyAway bus, which connects airport passengers to commuter rail.
```

Comment:
Notice the `?utm_source=Press+Release&utm_medium=Website&utm_id=FNG` part in the URL